group-aws/bestpractices.md

```diff
@@ -0,0 +1,19 @@
+# Best practices
```

Comment:
Maybe add in that the Working Group welcomes work to improve the best practices guidelines. 

If any of this checks can be automated AWS and Testing Working groups can do this.